rs/tests/system_tests.bzl

```diff
@@ -263,6 +268,50 @@ def system_test(
         flaky = flaky,
     )
 
+def system_test_nns(name, extra_head_nns_tags = ["system_test_nightly"], **kwargs):
+    """Declares a system-test that uses the mainnet NNS and a variant that use the HEAD NNS.
+
+    Declares two system-tests:
+
+    * One with the given name which uses the NNS from mainnet as specified by testnet/mainnet_revisions.json
```

Comment:
Doesn't `testnet/mainnet_revisions.json` specify the current NNS subnet _replica_ version? As I understand the git revision of mainnet NNS _canisters_ might be different

Reply:
Thanks for spotting that! Fixed.